src/main/java/bio/terra/app/controller/GlobalExceptionHandler.java

```diff
@@ -120,10 +120,11 @@ private ErrorModel buildErrorModel(Throwable ex) {
     }
 
     private ErrorModel buildErrorModel(Throwable ex, List<String> errorDetail) {
-        logger.error("Global exception handler: catch stack", ex);
+        StringBuilder combinedCauseString = new StringBuilder();
         for (Throwable cause = ex; cause != null; cause = cause.getCause()) {
-            logger.error("   cause: " + cause.toString());
+            combinedCauseString.append("cause: " + cause.toString() + ", ");
```

Comment:
Doesn't this duplicate the information already being logged? If the exception `ex` is included in the log operation, then the exception message and stack crawl for all causes should be printed.